Documentation/policy/visibility.rst

```diff
@@ -0,0 +1,52 @@
+.. only:: not (epub or latex or html)
+  
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _proxy_visibility:
+
+*******************
+L7 Proxy Visibility
+*******************
+
+While :ref:`monitor` provides introspection into datapath state, it only
+provides L3/L4 visibility if it is ran in an environment which is running
+with no policy enforcement at L7. To get more visibility into the application
```

Comment:
While technically true, I think that this would state the intentions a bit more clearly with the following?

```
While :ref:`monitor` provides introspection into datapath events, by default it will
only provide visibility into L3/L4 packet events. If :ref``l7_policy` is configured,
it can also provide visibility into L7 protocols, however this requires the full
policy for each selected endpoint to be written. To get more visibility into the
application without configuring a full policy...
```

Reply:
Changed to be quite similar to this, PTAL.